GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -472,11 +501,22 @@ public async void GetTeachingEventAsync_WithId_ReturnsMatchingEvent()
         [Fact]
         public async void GetTeachingEventAsync_WithReadableId_ReturnsMatchingEvent()
         {
-            var events = await SeedMockTeachingEventsAsync();
+            var events = await SeedMockTeachingEventsAndBuildingsAsync();
             var result = await _store.GetTeachingEventAsync(events.First().ReadableId);
 
             result.ReadableId.Should().Be(events.First().ReadableId);
             result.Building.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task GetTeachingEventBuildings_ReturnsAll()
+        {
+            var buildings = await SeedMockTeachingEventBuildingsAsync();
+            _mockCrm.Setup(m => m.GetTeachingEventBuildings()).Returns(buildings);

```

Comment:
We shouldn't need to mock the CRM here as the method loads buildings from the store